runtime/drivers/file/watcher.go

```diff
@@ -119,6 +119,24 @@ func (w *watcher) flush() {
 		return
 	}
 
+	for p, event := range w.buffer {
+		if !event.IsCreate {
+			continue
+		}
```

Comment:
If it skips the non-create case, what is it exactly this solves? I thought the issue was some directories not having create events when moved?

Reply:
The order of events is unreliable. Merging them was what fixed it earlier. This change moves it to flush.